srcpkgs/lua-lgi/template

```diff
@@ -0,0 +1,81 @@
+# Template file for 'lua-lgi'
+pkgname=lua-lgi
+version=0.9.2
+revision=4
+wrksrc="lgi-${version}"
+hostmakedepends="pkg-config"
+makedepends="lua-devel lua51-devel lua52-devel libglib-devel libffi-devel gobject-introspection gir-freedesktop"
+_depends="gir-freedesktop"
+depends="${_depends} lua"
+short_desc="Lua bindings for gnome/gobject using gobject-introspection library"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/pavouk/lgi"
+distfiles="https://github.com/pavouk/lgi/archive/$version.tar.gz"
+checksum=cfc4105482b4730b3a40097c9d9e7e35c46df2fb255370bdeb2f45a886548c4f
+
+_alternatives="
+ lgi:dump-typelib:/usr/bin/dump-typelib__luaver__
+"
+alternatives=${_alternatives/__luaver__/5.3}
+
+post_extract() {
+	mkdir -p lua5.1
+	mv * lua5.1 || true
+	cp -a lua5.1 lua5.2
+	cp -a lua5.1 lua5.3
+}
+
+do_build() {
+	for luaver in 5.1 5.2 5.3; do
+		cd ${wrksrc}/lua${luaver}
+		make LUA_VERSION="${luaver}" CC="$CC" AR="$AR" RANLIB="$RANLIB" CFLAGS="${_common_cflags} -I${XBPS_CROSS_BASE}/usr/include/lua${luaver}" ${makejobs}
```

Comment:
this doesn't look right at all

for one, you should stick to 80 columns, and also, what is `_common_cflags`? The `CFLAGS` should look more like `CFLAGS=$CFLAGS -I...`

you don't need to pass `AR` or `RANLIB` as that's never used, but you need to pass `LDFLAGS`

Reply:
```sh
make LUA_VERSION="${luaver}" \
    LDFLAGS="$LDFLAGS" \
    CFLAGS="$CFLAGS -I ${XBPS_CROSS_BASE}/usr/include/lua${luaver}"
```

Is this ok?

Reply:
yeah